bin/ch/ch.cpp

```diff
@@ -94,7 +94,12 @@ void __stdcall PrintUsage()
 
 void __stdcall PrintChVersion()
 {
-    wprintf(_u("%s version %d.%d.%d.0\n"), hostName, CHAKRA_CORE_MAJOR_VERSION, CHAKRA_CORE_MINOR_VERSION, CHAKRA_CORE_PATCH_VERSION);
+#if CHAKRA_CORE_VERSION_RELEASE
+    wprintf(_u("%s version %d.%d.%d.0\n"),
+#else
+    wprintf(_u("%s version %d.%d.%d.0-beta\n"),
```

Comment:
I think this is fine. @liminzhu should handle checking over the various documentation, but I think at the moment it's all in discussion and maintained by @obastemur unless I missed something being published.